data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaCustomConsumer.java

```diff
@@ -227,6 +227,7 @@ <T> void consumeRecords() throws Exception {
             topicMetrics.getNumberOfPollAuthErrors().increment();
             Thread.sleep(10000);
         } catch (RecordDeserializationException e) {
+            LOG.error("Deserialization error while doing poll()", e);
```

Comment:
I originally had a problem where I didn't set the AWS_SECRET_ACCESS_KEY in the data-prepper docker image and it was failing to connect to the instance of [motoserver](https://github.com/getmoto/moto) I was running.  The helpful error message in the stack trace wasn't being displayed as it was about the 3rd "caused-by" down in the trace.  I had to add this error message to see it.

I don't think this is ideal so I wanted some guidance on how you want me to handle this error so that it provides a meaningful message to a user for correcting things.